pkg/operator/ceph/cluster/controller.go

```diff
@@ -133,6 +162,37 @@ func (c *ClusterController) StopWatch() {
 // ************************************************************************************************
 // Add event functions
 // ************************************************************************************************
+func (c *ClusterController) onK8sNodeAdd(obj interface{}) {
+	newNode, ok := obj.(*v1.Node)
+	if !ok {
+		logger.Warningf("Expected NodeList but handler received %#v", obj)
+	}
+	for i := range newNode.Spec.Taints {
+		if newNode.Spec.Taints[i].Effect == "NoSchedule" {
+			logger.Debugf("Skipping -> Node %s is Unschedulable", newNode.Labels[apis.LabelHostname])
+			return
+		}
+	}
+
+	for _, cluster := range c.clusterMap {
+		if cluster.Spec.Storage.AnyUseAllDevices() == false {
+			logger.Debugf("Skipping -> Do not use all Devices")
+			continue
+		}
+
+		if valid, _ := k8sutil.ValidNode(*newNode, cluster.Spec.Placement.All()); valid == true {
+			logger.Debugf("Adding %s to cluster %s", newNode.Labels[apis.LabelHostname], cluster.Namespace)
+			err := cluster.createInstance(c.rookImage)
+			if err != nil {
+				logger.Errorf("Failed to update cluster in namespace %s. %+v", cluster.Namespace, err)
+			}
+		} else {
+			logger.Debugf("Host %s is not valid", newNode.Labels[apis.LabelHostname])
+		}
+		logger.Infof("Added %s to cluster %s", newNode.Labels[apis.LabelHostname], cluster.Namespace)
```

Comment:
I think including the node name and the operation (add) is important for the message.
This message will still be printed after an invalid host. Should there be a return after the "Host %s is not valid" message?